app/Services/UserService.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\User;
+
+class UserService
+{
+    public function createUser($data)
+    {
+        $user = User::create($data);
```

Comment:
we can avoid creating a new variable here. Just return the result of User::create($data);